sky/cloud_stores.py

```diff
@@ -98,6 +98,11 @@ def make_sync_file_command(self, source: str, destination: str) -> str:
         return ' && '.join(all_commands)
 
 
+class GCSPathNotFoundError(Exception):
+    """Custom exception to indicate that a GCS path does not exist."""
+    pass
```

Comment:
Instead of creating a new exception, can we use `sky_exceptions.StorageBucketGetError` and put path not found in the error message?